virt/virt-2-5-release-notes.adoc

```diff
@@ -0,0 +1,198 @@
+
+include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
+// This line is attached to the above `virt-what-you-can-do-with-virt` module.
+// It is included here in the assembly because of the xref ban.
+You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+//Commented out because for GA release we have special text that covers this info. But going forward, we will want to include the following:
+//include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+[id="virt-2-5-new"]
+== New and changed features
+//Placeholder for new content.
+
+//CNV-4904 Microsoft's SVVP certification applies to RHCOS workers. The content below from the 2.4 release notes seems to cover this already.
+
+* {VirtProductName} is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
+
+** Red Hat Enterprise Linux CoreOS 8 workers. In the Microsoft SVVP Catalog, they are named _Red Hat {product-title} 4 on RHEL CoreOS 8_.
+
+** Intel and AMD CPUs.
+
+//CNV-7293 - Use OLM stable channel for installation
+
+//CNV-7294 - Cluster admin can now influence node placement using new API on HCO CR
+
+//CNV-7160 - New virtctl commands to expose raw qemu guest agent data
+
+//CNV-7295 - Download virtctl binary from the CLI Tools page
+
+
+//CNV-4572-Guest-OS Should we still keep this info from the 2.4 release notes?
+[id="virt-guest-os-new"]
+=== Supported guest operating systems
+{VirtProductName} guests can use the following operating systems:
+
+* Red Hat Enterprise Linux 6, 7, and 8.
+* Microsoft Windows Server 2012 R2, 2016, and 2019.
+* Microsoft Windows 10.
+
+Other operating system templates shipped with {VirtProductName} are not supported.
+
+[id="virt-2-5-networking-new"]
+=== Networking
+//Placeholder for new content.
+
+//CNV-5263 - Supported bonding modes with nmstate
+
+[id="virt-2-5-storage-new"]
+=== Storage
+//Placeholder for new content.
+
+//CNV-7170 - CDI import of container disks is faster and more efficient
+
+//CNV-7168 - DataVolume and storage diagnostics
+
+//CNV-6824 - Offline virtual machine snapshots
+
+//CNV-6825 - Smart cloning with snapshots
+
+[id="virt-2-5-web-new"]
+=== Web console
+//Placeholder for new content.
+
+//CNV-7315 - Expose next run configuration in the UI
+
+//CNV-7318 - Open the vnc/serial console in a new window
+
+//CNV-7314 - Expose VM image upload in UI
+
+//CNV-7317 - Expose qemu guest agent data in the UI
+
+[id="virt-2-5-changes"]
+== Notable technical changes
+//Placeholder for new content.
+
+//CNV-7169 - Auto-clone base images when creating VM in UI
```

Comment:
7169 should be in the web console 'New' features
In it's place here should be CNV-7293 - OLM stable channel

Reply:
Done